.vale/styles/config/vocabularies/book/accept.txt

```diff
@@ -0,0 +1,2 @@
+HAL
```

Comment:
I was not aware of this feature! Its definitely annoying that acronyms allowed by one package still get the warning from other packages...

I think this is a good solution! We should probably also add RTIC